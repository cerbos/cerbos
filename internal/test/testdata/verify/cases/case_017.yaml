description: Resources testdata file not named resources.yaml
