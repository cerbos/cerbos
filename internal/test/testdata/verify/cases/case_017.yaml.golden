{
  "suites": [
    {
      "file": "udf_test.yaml",
      "name": "TestSuite",
      "summary": {
        "overallResult": "RESULT_ERRORED"
      },
      "error": "Failed to load the test suite: invalid test \"John and his leave request\": resource \"pending_leave_request\" not found",
      "description": "Tests for verifying something"
    }
  ],
  "summary": {
    "overallResult": "RESULT_ERRORED"
  }
}
